service/history/workflow/mutable_state_impl.go

```diff
@@ -576,6 +577,18 @@ func (ms *MutableStateImpl) GetNexusCompletion(ctx context.Context) (nexus.Opera
 	if err != nil {
 		return nil, err
 	}
+	// Create the link information about the workflow to be attached to fabricated started event if completion is
+	// received before start response.
+	startLink := temporalnexus.ConvertLinkWorkflowEventToNexusLink(&commonpb.Link_WorkflowEvent{
+		Namespace:  ms.namespaceEntry.ID().String(),
```

Comment:
This should be the namespace name, not ID. We don't have APIs that can interact with namespace IDs.